superset/superset/assets/src/uast/override.less

```diff
@@ -0,0 +1,77 @@
+@import '../../stylesheets/less/cosmo/variables.less';
+
+body,
```

Comment:
please run remove copies in `superset/superset` directory. You need to change only `srcd/superset`,
You can run `make clean` before commiting changes to avoid adding it in the future.

Reply:
Thank you, will do so.